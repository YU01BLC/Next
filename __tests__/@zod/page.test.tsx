import { render, screen } from "@testing-library/react";
import Page from "../../app/@zod/page";

describe("@zod/Pageコンポーネントscreenshot", () => {
  it("snapshot", () => {
    const { asFragment } = render(<Page />);
    expect(asFragment()).toMatchSnapshot();
  });
});

describe("@zod/Pageコンポーネント表示確認", () => {
  it("テキスト「zod」が表示されること", () => {
    render(<Page />);
    const headElement = screen.getByText("zod");
    expect(headElement).toBeInTheDocument();
  });
});
