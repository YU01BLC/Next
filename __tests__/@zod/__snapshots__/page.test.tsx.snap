// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@zod/Pageコンポーネントscreenshot snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-cv1rqq"
  >
    zod
  </div>
</DocumentFragment>
`;
